### DISCUSSION TITLE
建议每个单词都是固定的长度

### DISCUSSION THREAD

**Discussion by baixiaoyu2997** [cuixueshe/earthworm#244] [General]
![image](https://github.com/cuixueshe/earthworm/assets/9149999/e1e6c276-afca-4980-beb6-de9b8be4474c)
之前的版本是固定长度的，但是现在这样，我根本没办法回想，直接就猜出来了

**Comment by fengstats** [comment]
hello~ 我们考虑到了这个问题

所以我们在用户设置中提供了一个开关用于选择是否需要显示对应单词长度，关闭即可

不过注意需要先登录哦~ 😊

![1710246824323](https://github.com/cuixueshe/earthworm/assets/48991003/8a7a7781-b6be-4416-b738-fa8083a041d5)

**Reply by fengstats** [reply]
@baixiaoyu2997

**Reply by baixiaoyu2997** [reply]
非常感谢~~~
